renderer/components/preferences/categories/general.js

```diff
@@ -184,6 +185,18 @@ class General extends React.Component {
         >
           <Button tabIndex={tabIndex} title="Choose" onClick={pickKapturesDir}/>
         </Item>
+        <Item
+          key="lossyCompression"
+          parentItem
+          title="Lossy compression"
+          subtitle="Compress GIFs with lossy compression (this will result in minor quality degredation and smaller file size)"
```

Comment:
How about this?

```suggestion
          title="Lossy GIF compression"
          subtitle="Smaller file size for a minor quality degradation."
```

Reply:
Haha yeah that's great. I was struggling with this. 😅